### PR TITLE
Add homepage content and styles

### PR DESCRIPTION
**Pull request by magwalk**
screen shot 2016-09-20 at 11 30 24 am
screen shot 2016-09-20 at 11 30 33 am

### CODE REVIEW

**Review by cwebberOps — APPROVED** [review]
Looks great!

File: www/source/index.html.slim [hunk 1]
```diff
@@ -1,108 +1,80 @@
...
+      .columns.large-3.medium-6
+        img.icon-block--img src="/images/home/test-icon.png"
+        h3.icon-block--heading Test Locally or Remotely
+        p
+          | InSpec provides a local agent, as well as full remote testing support.
+      .columns.large-3.medium-6
+        img.icon-block--img src="/images/home/free-icon.png"
+        h3.icon-block--heading Free to Run Anywhere
+        p
+          | InSpec It is a language that can easily express freedom to run anywhere.
```
Comment (TrevorBramble) [inline www/source/index.html.slim]: > InSpec ~~It~~ is a

File: www/source/index.html.slim [hunk 2]
```diff
@@ -1,108 +1,80 @@
...
+      .columns.large-6
+        img.code-snippet--img[src="/images/home/code-2.svg" onerror="this.src='/images/home/code-2.png'"
+          alt="Code Snippet"]
+      .columns.large-6.code-snippet--panel.second
+        small.code-snippet--description
+          strong Solve your infrastructure testing needs simply and efficiently.
+        h3.code-snippet--heading Test the desired state:
+        p.code-snippet--text Match your infrastructure to your expectations.
+        h3.code-snippet--heading Use simple and expressive code:
+        p.code-snippet--text These tests are easy to understand by anyone, reducing friction.
```
Comment (TrevorBramble) [inline www/source/index.html.slim:69]: I'd suggest dropping "these" but that's a copywriter call.

File: www/source/index.html.slim [hunk 3]
```diff
@@ -1,108 +1,80 @@
...
+        h3.code-snippet--heading Test any system anywhere:
+        p.code-snippet--text
+          | Runs locally and remotely. Supports all major operating systems
+            and configurations.
+        h3.code-snippet--heading Extensible:
+        p.code-snippet--text Easily create custom resources and share them.
+    hr.home-divider
+    h2.main-subhead.text-center The Guiding Principles of InSpec
+    p Lorem ipsum...
```
Comment (TrevorBramble) [inline www/source/index.html.slim:78]: Intended?

**Comment by chris-rock** [comment]
Love it.

File: www/source/index.html.slim (hunk 3 above)
Comment (magwalk) [reply www/source/index.html.slim:78]: Yep - we're waiting on copy to be updated later.

**Pull request merged**